src/lib/mina_base/zkapp_precondition.ml

```diff
@@ -1105,6 +1105,39 @@ module Protocol_state = struct
           Epoch_data.Poly.t )
         Poly.t
     end
+
+    let epoch_data_deriver obj =
```

Comment:
yeah, this is because I'm stacked on https://github.com/MinaProtocol/mina/pull/11639 which is a forked repo.. will rebase or wait for that PR to be merged

Reply:
ok, fixed